dynamodb/tests/hasProperties.test.w

```diff
@@ -0,0 +1,36 @@
+bring expect;
+bring aws;
+bring cloud;
+bring ex;
+bring "../dynamodb-types.w" as dynamodb_types;
+bring "../dynamodb.sim.w" as dynamodb_sim;
+bring "../dynamodb.w" as dynamodb;
+
+let tableWithBillingModeProvisioned = new dynamodb.Table({
+  billingMode: dynamodb_types.BillingMode.PROVISIONED,
+  attributes: [
+      { name: "id", type: "S" },
+  ],
+  hashKey: "id",
+});
+
+let simClient = dynamodb_sim.Util.createClient({
+  endpoint: "http://localhost:8000",
+  region: "us-west",
+  credentials: {
+    accessKeyId: "accessKeyId",
+    secretAccessKey: "secretAccessKey",
+  },
+});
+
+test "Sim: table with billing mode `provisioned`" {
+
+  let tableDescription = simClient.describeTable({
+    TableName: tableWithBillingModeProvisioned.tableName,
+  });
+
+  expect.equal(
+    tableDescription.billingMode,
+    dynamodb_types.BillingMode.PROVISIONED
+  );
+}
```

Comment:
I think this test will fail if you try running it on the `tf-aws` platform since it's directly calling into the `dynamodb_sim` implementation. If we want to scope this test just to run on the simulator, you can wrap all of it in a block like:

```
bring util;

if util.env("WING_TARGET") == "sim" {
  // resources...
  // tests ...
}
```